pkg/dispers/dispers/metadata.go

```diff
@@ -0,0 +1,141 @@
+package dispers
+
+import (
+	"errors"
+	"os"
+	"time"
+
+	"github.com/cozy/cozy-stack/pkg/couchdb"
+	"github.com/cozy/cozy-stack/pkg/couchdb/mango"
+	"github.com/cozy/cozy-stack/pkg/prefixer"
+)
+
+// Only the Conductor should save metadata
+var prefix = prefixer.ConductorPrefixer
+
+// Metadata are written on the confuctor's database. The querier can read those Metadata to know his query's state
+type Metadata struct {
+	Start       time.Time `json:"start,omitempty"`
+	End         time.Time `json:"end,omitempty"`
+	Duration    string    `json:"duration,omitempty"`
+	Host        string    `json:"host,omitempty"`
+	Tags        []string  `json:"tags,omitempty"`
+	Outcome     bool      `json:"outcome,omitempty"`
+	Error       string    `json:"error,omitempty"`
+	Name        string    `json:"name,omitempty"`
+	Description string    `json:"description,omitempty"`
+	Output      string    `json:"output,omitempty"`
+}
+
+// NewMetadata returns a new Metadata object
+func NewMetadata(name string, description string, tags []string) Metadata {
+	now := time.Now()
+	host, _ := os.Hostname()
```

Comment:
This will be the hostname of the conductor? What is it for?

Reply:
Nein. 

I was not clear about what I imagined metadata to be. I thought the PR's description and [this description)(https://github.com/tintinmar1995/cozy-stack/tree/enclaves-docs/docs#keep-a-look-on-the-query) was clear enought but obviously not.

We make sure with Zoom we want Metadata/Logs to do the same job and I will do what it needs to be done to have a clear documentation about Metadata/Logs.

Reply:
> This will be the hostname of the conductor? 

If we consider a DISPERS network with two server A and B. `CI from A` will instanciate the Metadata/Log and Close it. A will send Metadata in the same time than `OutputCI`. `Conductor from B` will have to save it in its database with the push method. 

The Querier will have a dashboard that display the metadata/log in a pretty way with propal, material UI, gif, color, ....